lib/foreman_inventory_upload/async/generate_all_reports_job.rb

```diff
@@ -5,6 +5,13 @@ class GenerateAllReportsJob < ::Actions::EntryAction
       include ForemanInventoryUpload::Async::DelayedStart
 
       def plan
+        if ForemanRhCloud.with_local_advisor_engine?
+          logger.debug(
+            'The scheduled process is disabled due to the "use_local_advisor_engine"
+            SETTING being set to true.'
```

Comment:
Instead of a debug log, may be more useful to put this in the task's output. like 
```
output[:status] = _('The scheduled process is disabled because this Foreman is configured with the use_local_advisor_engine option.')
```

Reply:
also please note my suggested wording ;)